tox.ini

```diff
@@ -55,8 +55,9 @@ deps =
     pytest-factoryboy
     pytest-mock
     pytest-cov
+    pytest-memprof
 commands =
-    pytest --cov=rgd --cov=core --cov=geodata {posargs}
+    pytest --cov=rgd --cov=core --cov=geodata --durations=0 {posargs}
```

Comment:
Consider adding additional CLI options to the `[pytest] addopts` section if you want them to always be included.

Reply:
Done